core/prelude/operators/comparison.carbon

```diff
@@ -20,3 +20,9 @@ interface Ordered {
   fn Greater[self: Self](other: Self) -> bool;
   fn GreaterOrEquivalent[self: Self](other: Self) -> bool;
 }
+
+// Equality comparison for `bool`.
```

Comment:
Can you add a note about why this is here instead of in prelude/types/bool? i.e., to discourage similar additions here for types that don't need it.

Reply:
Done.